### DISCUSSION TITLE
Doesn't work well with DuckDuckGo

### DISCUSSION THREAD

**Discussion by MarjaE2** [plateaukao/einkbro#170] [Ideas]
I'm getting frustrated with the combination of EinkBro and it's default browser, DuckDuckGo.

1. DuckDuckGo uses tiles for each result, and interprets any click or tap on the tile as a choice to select that result.
2. EinkBro hides the outline for the tiles. So if I don't want to go to a result page, and don't tap on the underlined link, I still often end up on that result page anyway...
3. As a page loads or reloads, it often jumps about, and if it reloads, EinkBro then returns to its former place. So if I try to tap one place, but the page jumps underneath, I end up in another place.
4. As the on-screen keyboard opens and closes, the tap areas to page up or down jump too.

**Comment by plateaukao** [comment]
1., 2. From my devices, if I use DuckDuckGo as search engine, the outline of tiles are always there. Could you take screenshot to show what it looks like on your device? I don't quite get what you mean that "you still often end up on that result page".
3. please give me an example, like what you search, and at what position you tried to reload, and EinkBro returns to its former place. On my devices, this does not happen.  I need reproduce steps to check what might go wrong.
4. Yes, it's intentionally implemented this way. When keyboard shows up, all the web content is shifted up too. Accordingly, the touch area is shifted too.

**Reply by MarjaE2** [reply]
1. The screenshot shows color, etc. which the screen didn't show. But it still doesn't show outlines around most results. Maybe it depends on Duck settings.

![Screenshot_20221229-135322](https://user-images.githubusercontent.com/12875862/210040579-6f71e573-daba-4f0f-adae-f860a89081ed.png)

3. I retried that search, Bratianu OPetlura Denikin 1919, paged down a few times. If I used "More Results," paged down again, and reladed, it would reload at "More Results." Otherwise if I paged down, it would reload some distance down the page.

**Comment by plateaukao** [comment]
The reason why search results are without outline is because it looks like duckduckgo will check screen size to decide whether to show outlines or not. EinkBro does not do anything to hide it. Also, maybe turning off Destop mode may help.

As for reloading issue. I found it has same behavior on Chrome too. So, it's duckduckgo implementation. You may need to report to Duckduckgo instead.

**Comment by Eloitor** [comment]
I use https://lite.duckduckgo.com/
